Sources/VLCMediaPlayer.m

```diff
@@ -1328,6 +1360,34 @@ - (void *)libVLCMediaPlayer
     return _playerInstance;
 }
 
+- (BOOL)startRecordingAtPath:(NSString *)path
+{
+    return libvlc_media_player_record(_playerInstance, YES, [path UTF8String]);
+}
+
+- (BOOL)stopRecording
+{
+    return libvlc_media_player_record(_playerInstance, NO, nil);
+}
+
+- (BOOL)muxSubtitleFile:(NSString *)srtPath toMp4File:(NSString *)mp4Path outputPath:(NSString *)outputPath
+{
+    //check if mp4 file is valid, srt is valid
+
+    libvlc_media_t* p_media = libvlc_media_new_path([_privateLibrary instance], [mp4Path UTF8String]);
+
+    NSString *option = [NSString stringWithFormat:@":sout=#transcode{vcodec=h264,width=720,height=480,venc=avcodec{codec=h264_videotoolbox},acodec=mpga,ab=128,channels=2,samplerate=44100,soverlay}:file{dst='%@',mux=mp4}", outputPath];
```

Comment:
Do we want to check whether outputPath exists or fail silently?
Additionally, consider renaming the variable as "option" is a bit generic.